arch/arm64/start.S

```diff
@@ -32,6 +32,11 @@ attr                    .req x27
 FUNCTION(_start)
 .globl arm_reset
 arm_reset:
+    /* keep track of the boot EL */
```

Comment:
This is tricky, since all cpus enter via this hook. Should probably only save it when on cpu 0, though probably all of the other cpus enter at the same EL so mostly likely it works. 

Can you add a branch here that tests for cpu 0 (mpidr with AFF0-3 masked off) and only saves it for that one? Or, save it into a register and then only conditionally write it out in the code later on that is known to be on cpu 0 (around line 73).

Reply:
Done.